packages/services/f-feature-management/src/index.js

```diff
@@ -0,0 +1,34 @@
+const logValueRequest = (func, key, value) => {
+    console.log(`${func.name}: ${key} => ${value}`); // es-lint-disable
+}
+
+function getBooleanValue (key) {
+    logValueRequest(getBooleanValue, key, true);
+    return true; // TODO: implementation
+}
+
+
+function getIntegerValue (key) {
+    logValueRequest(getIntegerValue, key, true);
+    return 0; // TODO: implementation
+}
+
+
+function getStringValue (key) {
+    logValueRequest(getStringValue, key, true);
+    return ''; // TODO: implementation
+}
+
+export default function (scope) {
+    function fullKey (key) {
+        return (scope && `${scope}::${key}`) || key;
+    }
+    function forward (fn, key) {
+        return fn(fullKey(key));
+    }
+    return {
+        getBooleanValue: key => forward(getBooleanValue, key),
+        getIntegerValue: key => forward(getIntegerValue, key),
+        getStringValue: key => forward(getStringValue, key)
+    };
```

Comment:
I'd agree with @xander-marjoram on this. My preference on initial glance would be to refactor fullKey to ES6 syntax and git rid of forward entirely, like so:
```suggestion
    const fullKey = key => scope
        // ternary preferred over && and || operators, and break to indented multi-line for readability
        ? `${scope}::${key}`)
        : key;

    return {
        getBooleanValue: key => getBooleanValue(fullKey(key)),
        getIntegerValue: key => getIntegerValue(fullKey(key)),
        getStringValue: key => getStringValue(fullKey(key))
    };
```